README.md

```diff
@@ -1,16 +1,32 @@
-# 📦 Axios Module
+[![@nuxtjs/axios](https://axios.nuxtjs.org/preview.png)](https://axios.nuxtjs.org)
 
+# @nuxtjs/axios
 
 [![npm version][npm-version-src]][npm-version-href]
 [![npm downloads][npm-downloads-src]][npm-downloads-href]
-[![Checks][checks-src]][checks-href]
+[![Github Actions CI][github-actions-ci-src]][github-actions-ci-href]
 [![Codecov][codecov-src]][codecov-href]
+[![License][license-src]][license-href]
 
-> Secure and Easy <a href="https://github.com/mzabriskie/axios">Axios</a> integration with Nuxt.js
+> Secure and easy [Axios](https://github.com/axios/axios) integration for [Nuxt](https://nuxtjs.org).
 
-📖 [**Read Documentation**](https://axios.nuxtjs.org)
+- [✨ &nbsp;Release Notes](https://axios.nuxtjs.org/releases)
+- [📖 &nbsp;Documentation](https://axios.nuxtjs.org)
 
-## Development
+## Features
+
+- Automatically set base URL for client & server side
+- Exposes `setToken` function to `$axios` for authentication tokens
+- Automatically enables `withCredentials` when requesting to base URL
+- Proxy request headers in SSR (Useful for auth)
```

Comment:
Proxy headers is to ensure requests are isomorphic between client and server not necessarily auth